src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -111,72 +88,14 @@ public void run() {
-                                            ntf -> Collections.singleton(
-                                                new MapEntry<>(
-                                                    ntf.subject().getString("title"),
-                                                    String.format("click:notification#%s", ntf.tid())
-                                                )
-                                            ),
-                                            nts
-                                        )
-                                    );
-                                }
+                                    new TkNotificationsCommand(this.data)
                             ),
                             new FkCallbackQuery(
                                 Pattern.compile("click:notification#(?<tid>[A-Za-z0-9]+)"),
-                                upd -> {
-                                    final GhUser user = new PgUsers(this.data).user(upd.getCallbackQuery().getMessage().getChat()).github();
-                                    final GhThread thread = user.thread(upd.getCallbackQuery().getData().split("#")[1]);
-                                    final Issue issue = new ThreadIssue(user.github(), thread);
-                                    return new RsText(
-                                        new FormattedText(
-                                            "[#%d](%s) - %s\n\n%s",
-                                            issue.number(),
-                                            String.format("https://github.com/%s/issues/%d", issue.repo().coordinates(), issue.number()),
-                                            new Issue.Smart(issue).title(),
-                                            new Joined(
-                                                new TextOf("\n\n"),
-                                                new Mapped<>(
-                                                    cmt -> new FormattedText(
-                                                        "[@%s](https://github.com/%s) > %s",
-                                                        cmt.author().login(),
-                                                        cmt.author().login(),
-                                                        cmt.body()
-                                                    ),
-                                                    new Mapped<>(
-                                                        Comment.Smart::new,
-                                                        issue.comments().iterate(new Date(thread.lastRead().toEpochMilli()))
-                                                    )
-                                                )
-                                            )
-                                        )
-                                    );
-                                }
+                                    new TkNotificationsFallback(this.data)
```

Comment:
@samihibrahim it's not correct class name, this class handles button clicks returned by `/notifications` command and returns single thread from notification, so it's better to rename it to `TkThread` or similar.

Reply:
@g4s8 Fixed.